my-app/src/components/ResultPage/ResultPage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useEffect,useState} from 'react'
+import Menu from '../Menu/Menu'
+import Title from '../Title/Title'
+import { StyledContainerResult } from './styled'
```

Comment:
Put all of them in a single import:

`import { StyledContainerResult, StyledImagePost, StyledContentResult ... } from './styled'`